docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-metrics-exporter.md

```diff
@@ -56,6 +56,8 @@ Assign an export configuration to the specified cluster.
 
 ### create
 
+Deprecated 2024-11-30 and will be supported until 2025-2-28. Use `integration create` instead.
```

Comment:
Do you think these could use some kind of highlighting to make them a little more visible?